app/models/user.rb

```diff
@@ -96,10 +97,11 @@ class User < ActiveRecord::Base
 
   has_many :notifications, dependent: :destroy
   has_many :comments, dependent: :destroy
-  has_many :attachments
+  has_many :attachments, dependent: :destroy
```

Comment:
I'm not leaving comments. Deactivate is where we keep user data.. delete is where we do our best to remove everything and leave no dangling references. It's a work in progress, we do not do any user deletion right now